src/test/reducerTests/integrations.test.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable no-unused-expressions */
+import React from 'react';
```

Comment:
- [ ] The linters are there to enforce code quality and should not be disabled. Kindly enable the linters and fix the issues raised.